File: src/main/java/toast/client/commands/cmds/ClearChat.java
package toast.client.commands.cmds;

import toast.client.commands.Command;
import toast.client.utils.Logger;

public class ClearChat extends Command {
    public ClearChat(){ super("clearchat", "Clears all messages in chat", false, "clearchat"); }

    @Override
    public void run(String[] args) throws InterruptedException {
        if(mc.player == null) return;
        if(mc.inGameHud.getChatHud() != null) {
            mc.inGameHud.getChatHud().clear(true);
            Logger.message("Cleared chat", Logger.SUCC);
        }else { Logger.message("Fuck I don't know chat hud is null ¯\\_(ツ)_/¯", Logger.ERR); }
    }
}

File: src/main/java/toast/client/commands/cmds/ClearChat.kt
package toast.client.commands.cmds

import toast.client.ToastClient
import toast.client.commands.Command
import toast.client.utils.MessageUtil

/**
 * Command to wipe all messages from the in-game chat HUD
 */
class ClearChat : Command("ClearChat", "${ToastClient.cmdPrefix}clearchat", "Clears all messages in chat", false, "clearchat") {
    @Throws(InterruptedException::class)
    override fun run(args: Array<String>) {
        if (mc.player == null) return
        if (mc.inGameHud.chatHud != null) {
            mc.inGameHud.chatHud.clear(true)
            MessageUtil.sendMessage("Cleared chat!", MessageUtil.Color.GREEN)
        } else {
            MessageUtil.sendMessage("Fuck I don't know chat hud is null ¯\\_(ツ)_/¯", MessageUtil.Color.GRAY)
        }
    }
}